data/mainnet/addresses.json

```diff
@@ -1,2453 +1,2488 @@
 {
```

Comment:
Can you reset this file to the original formatting, so we don't get 5k lines of diff?

Reply:
Fixed!